lib/futurism.rb

```diff
@@ -17,16 +18,23 @@ module Futurism
   autoload :Helpers, "futurism/helpers"
 
   mattr_accessor :skip_in_test, default: false
+  mattr_accessor :instrumentation, default: false
+  mattr_accessor :logger
 
   mattr_writer :default_controller
   def self.default_controller
     (@@default_controller || "::ApplicationController").to_s.constantize
   end
 
+  def self.skip_in_test?
```

Comment:
I do have a slight reservation of this rename, because it's technically a breaking change. I would rather not do a major version bump right now, could you revise that to `self.skip_in_test` again?

Reply:
Sorry I was confused because I was on my way out the door ... so indeed no existing usage of `skip_in_test` is broken.

I'll double check that